schema/transportation/segment.yaml

```diff
@@ -221,6 +221,31 @@ properties:
                     items: { "$ref": "#/$defs/propertyDefinitions/lane" }
                     minItems: 1
               minItems: 1
+        signposts:
+          description: Describes signpost along the a segment
+          type: array
+          items:
+             allOf:
+               - "$ref": "#/$defs/propertyContainers/directionContainer"
+             unevaluatedProperties: false
+             type: object
+             properties:
+               destination:
+                 type: object
+                 required: [segmentId, signTextType, signText]
+                 allOf:
+                   - "$ref": "#/$defs/propertyContainers/directionContainer"
+                 properties:
```

Comment:
How do we know at what point along the segment the signs exist?

Reply:
@vbogdana had a similar question. Agree this is a gap.